build.gradle

```diff
@@ -66,6 +66,17 @@ sourceSets {
   }
 }
 
+idea {
```

Comment:
this marks the integration test, smoke test and func tests modules as test modules in IntelliJ